pkg/image/builder.go

```diff
@@ -102,16 +102,22 @@ func GetPrivateImages(upstreamDir string, checkedImages map[string]ImageInfo, al
 				return err
 			}
 
-			if info.IsDir() && (useHelmInstall[info.Name()] == true) {
-				return filepath.SkipDir
-			}
-
 			if info.IsDir() {
+				chartName, err := filepath.Rel(upstreamDir, path)
+				if err != nil {
+					logger.Debugf("Failed to remove prefix from %s: %v", path, err)
+					return nil
+				}
+				chartName = filepath.Join(parentHelmChartPath, chartName)
```

Comment:
shouldn't `filepath.Rel(upstreamDir, path)` already have the parent helm chart path?

Reply:
each helm chart is treated as a separate base, so starting root will be nested inside one or more charts, and the map contains values relative to top level.

Reply:
ah that's why i asked if a double nested subchart would work here